code/style.css

```
@@ -1,5 +1,333 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@import url("https://fonts.googleapis.com/css2?family=Cookie&family=Domine&family=Open+Sans:wght@400;700&display=swap");
 
 body {
-  background-color: lightblue;
+  margin: 0 auto;
+  max-width: 1440px;
+}
+/* evt. Header einfügen mit background-size: cover, und height 2527 beim Bild Save the date */
+
+h1 {
+  color: rgb(161, 13, 40);
+  font-size: 36px;
+}
+
+header {
+  background-image: url("./images/Front\ 1.jpg");
+  height: 500px;
+  background-size: cover;
+}
+.top-heading {
+  color: rgb(124, 6, 51);
+  font-size: 60px;
+  font-weight: bolder;
+  margin: 0;
+  text-align: center;
+  padding-top: 50px;
+  font-family: "Cookie", cursive;
+}
+.section-image {
+  width: 100%;
+}
+.top-image {
+  max-width: 5400px;
+  background-position: top;
+  width: 100%;
+  max-height: 400px;
+  overflow: hidden;
+  /* es braucht nur width 100% damit das Bild wieder sauber dargestellt wird */
+}
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: "Open Sans", sans-serif;
+}
+
+nav a {
+  display: inline-block;
+  padding: 10px 15px;
+  background: rgb(161, 13, 40);
+  color: rgb(0, 0, 0);
+  font-weight: semi-bold;
+  text-decoration: none;
+  border-radius: 3px;
+  margin: 0 5px;
+}
+
+nav a:hover {
+  background: rgb(161, 13, 40);
+  color: white;
+}
+h2 {
+  text-align: center;
+  background-color: rgb(161, 13, 40, 0.7);
+  padding: 20px;
+  margin: 0;
+  font-family: "Domine", serif;
+}
+h3 {
+  text-align: center;
+  color: rgb(161, 13, 40, 0.9);
+  padding: 15px;
+  margin: 0px;
+  font-family: "Domine", serif;
+}
+h4 {
+  text-align: center;
+  padding: 10px;
+  margin: 0px;
+  font-family: "Domine", serif;
+}
+p {
+  text-align: center;
+  padding-left: 20px;
+  padding-right: 20px;
+  font-family: "Open Sans", sans-serif;
+}
```

Some of your texts are very long and hard to read if you don't wrap it. The optimal line width for text is between 60 - 70 characters. So you could set `max-width: 70ch;` to increase readability and then `margin: auto`. to keep the text centered.
```suggestion
p {
  text-align: center;
  max-width: 70ch;
  margin: auto;
  padding-left: 20px;
  padding-right: 20px;
  font-family: "Open Sans", sans-serif;
}
```